packages/scalecube-discovery/src/helpers/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const isNodeEnv = () => typeof window === 'undefined';
```

Comment:
`isNodeEnv` function doesn't check for NodeJS, it's checking for `window`
actually, it seem obsolete